evaluator/src/main/kotlin/SourceTreeResolver.kt

```diff
@@ -47,7 +47,7 @@ class SourceTreeResolver private constructor(getRepositoryRoot: () -> File) {
         fun forLocalDirectory(dir: File) = SourceTreeResolver { dir }
     }
 
-    val rootDir: File by lazy {
+    val rootDir by lazy {
```

Comment:
I'd prefer to keep the explicit type for readability.

Reply:
Fine 🤷🏻‍♂️ I've dropped that commit.